### PR TITLE
OC-15058:fix the issue of no longer throwing a warning message when r…

### PR DESCRIPTION
**Pull request by Taoli2018**
…esponse options are changed between versions

### CODE REVIEW

File: web/src/main/java/org/akaza/openclinica/control/admin/CreateCRFVersionServlet.java [hunk 1]
```diff
@@ -686,7 +686,41 @@ public ResponseOptionBean hasDifferentOption(ResponseSetBean oldRes, ResponseSet
...
         if (oldOptions.size() != newOptions.size()) {
             // if the sizes are different, means the options don't match
-            return null;
+        	//OC-15058, but still need to compare the "overlapped" part
+        	int smallerSize =0;
+        	if(oldOptions.size() < newOptions.size()) {
+        		smallerSize = oldOptions.size();
+        	}else {
+        		smallerSize = newOptions.size();
+        	}
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/control/admin/CreateCRFVersionServlet.java:695]: If the sizes are different, should we throw an error?

Comment (svadla-oc) [reply web/src/main/java/org/akaza/openclinica/control/admin/CreateCRFVersionServlet.java:695]: What happens if the order of the values change?

Comment (Taoli2018) [reply web/src/main/java/org/akaza/openclinica/control/admin/CreateCRFVersionServlet.java]: Get  more feedback about the requirements from CS team:
If the sizes are different, should we throw an error? ---- NO, still allow to process.
What happens if the order of the values change? --- Order doesn't matter as long as the coded values are kept the same
Did more code changes after reconsider above feedback.

**Comment by svadla-oc** [comment]
👍

**Pull request merged**